README.rst

```diff
@@ -203,7 +203,7 @@ page or ask us at our active Gitter channel, where we have maintainers from
 all over the world. We appreciate any help!
 
 We also have a
-`newcomer guide <http://coala.readthedocs.io/en/latest/Developers/Newcomers_Guide.html>`__
+`newcomer guide <http://api.coala.io/en/latest/Developers/Newcomers_Guide.html>`__
 to help you get started by fixing an issue yourself! If you get stuck anywhere
```

Comment:
We prefer the short URL, (coala.io/Newcomers).
If the location of the Newcomers docs change int the future, we can easily change the redirection instead of changing all instances of it in the documentation.
Hope this makes sense. Thanks :)